cmd/commands/restore.go

```diff
@@ -32,6 +32,7 @@ var RestoreCmd = &cobra.Command{
 		verifyOperatorPodIsRunning(cmd.Context(), clientSets)
 	},
 	Run: func(cmd *cobra.Command, args []string) {
-		restore.RestoreCrd(cmd.Context(), clientSets, operatorNamespace, cephClusterNamespace, crds.CephRookIoGroup, crds.CephRookResourcesVersion, args)
+		customResources := []restore.CustomResource{}
+		restore.RestoreCrd(cmd.Context(), clientSets, operatorNamespace, cephClusterNamespace, crds.CephRookIoGroup, crds.CephRookResourcesVersion, "rook-ceph-operator", customResources, args)
```

Comment:
do we need to update the doc cmd with this?

Reply:
No change needed for users of the kubectl plugin, since the rook operator is still being stopped as before. It's only API consumers that have a change, but we don't have documentation for the API.